hw/ip/gpio/data/gpio.hjson

```diff
@@ -110,7 +110,9 @@
       act:     "rcv",
       package: "",
       desc:    '''
-               This signal is pulsed high by the power manager after reset in order to sample the HW straps.
+               The strap enable signal is used to make the snapshot of the gpio_in.
+               It can trigger only one-time after the power-on reset.
```

Comment:
I don't think this is quite true: that sounds like a constraint on the behaviour of the port. I think you want to say something like "This only has an effect the first time it goes high".